### ISSUE TITLE
delaysContentTouches on scrollView

### ISSUE THREAD

**Issue by enhancient** [uias/Pageboy#58]
Hi,

Any chance of adding a way to set the delaysContentTouches variable of the scrollView in PageboyViewController? It doesn't seem that the scrollView is publicly accessible and I don't see a way I can override for child ViewControllers.

Something like adding:

```
public var delaysContentTouches: Bool = true {
        didSet {
            self.pageViewController.scrollView?.delaysContentTouches = delaysContentTouches

        }
    }

```
Thanks for a great library!

**Comment by msaps** [comment]
@enhancient just added this to `PageboyViewController` via a `delaysContentTouches` property that maps directly to the inner scroll view. 

Available from `v1.0.9` 😄

**Comment by enhancient** [comment]
@msaps Thanks!
